.eslintrc

```diff
@@ -1,3 +1,3 @@
 {
-  "extends": "airbnb"
+  "extends": "./index.js"
```

Comment:
This has been tested. It works as expected. You'll see an example of it under [Using a Configuration File](http://eslint.org/docs/user-guide/configuring) in the yarn docs.